app/models/grantee.rb

```diff
@@ -9,6 +9,11 @@ class Grantee
   def initialize(hses_grantee:)
     @id = hses_grantee[:id]
     @attributes = hses_grantee[:attributes].with_indifferent_access
+    @links = hses_grantee[:links].with_indifferent_access
+  end
+
+  def hses_link
+    links["hses"]
```

Comment:
[issue] so with the above change, this will become `links[:html]`

i'm curious about whether, since we're pulling so many things out of the hses_grantee object, if we should just be saving that as an attr_reader, and then having individual methods to pull the fields out of them. thoughts?